ib-cli/installationiso/data/install-rhcos-and-restore-seed.sh

```diff
@@ -6,28 +6,26 @@ seed_image=${1:-$SEED_IMAGE}
 seed_version=${2:-$SEED_VERSION}
 installation_disk=${3:-$INSTALLATION_DISK}
 lca_image=${4:-$LCA_IMAGE}
+extra_partition_start=${5:-$EXTRA_PARTITION_START}
+extra_partition_label=varlibcontainers
 
 authfile=${AUTH_FILE:-"/var/tmp/backup-secret.json"}
 pull_secret=${PULL_SECRET_FILE:-"/var/tmp/pull-secret.json"}
 
 coreos-installer install ${installation_disk}
 
+# Create new partition for /var/lib/containers
+sfdisk ${installation_disk} <<< write
+sgdisk --new 5:$extra_partition_start --change-name 5:$extra_partition_label ${installation_disk}
```

Comment:
Why there's `5` everywhere?

Reply:
There are 4 standard partitions for RHCOS, we are creating a new one, and so it will be the 5th one. Thus the 5

Reply:
got it, create a var and use it just to make the code clear
e.g.:
`PRECACHCING_PARTITION_NUMBER=5`

Reply:
done :)